packages/react/src/components/Autofill/Autofill.tsx

```diff
@@ -326,7 +326,8 @@ export class Autofill extends React.Component<IAutofillProps, IAutofillState> im
       newValue = onInputChange?.(newValue, composing) || '';
     }
 
-    this.setState({ inputValue: newValue }, () => onInputValueChange?.(newValue, composing));
+    this.setState({ inputValue: newValue });
```

Comment:
What's the reason for this timing change? Could have some unintended side effects since the component won't be fully updated yet.

Reply:
I was trying to workaround an issue where sometimes we get two calls to _onInputChange for one entered key with IME keyboard.  It is only in a controlled component situation where value prop is bound to some function.  This causes us to sometimes see double characters.  The more I investigate this the more I believe the issue is outside Autofill and has something to do with React input sending multiple onChange.  It feels like a bug but I'm not sure if it isn't by design.  I've filed issue #19054 to track.

Reply:
Interesting, thanks for the investigation! Have you checked to see if anyone has reported a similar issue to React itself, or if you can find any other references to it online (stack overflow questions, other github issues, etc)?

Reply:
Actually I'll follow up about that on the issue :)